build/build-dt-report-resources.js

```diff
@@ -26,8 +26,9 @@ function writeFile(name, content) {
   fs.writeFileSync(`${distDir}/${name}`, content);
 }
 
+fs.mkdirSync(distDir, {recursive: true}); // Ensure dist is present, else rmdir will throw
 fs.rmdirSync(distDir, {recursive: true});
-fs.mkdirSync(distDir);
```

Comment:
Instead we can use fs.rmSync with force: true and recursive: true.

Reply:
`rm`/`rmSync` aren't supported until Node 14

Reply:
`rmSync` is new in node 14. (not in 12). :/

`fsPromises.rm` has force, but will require a bigger diff in this file.